ui/src/testing/stubs/cdr-versions-api-stub.ts

```diff
@@ -86,9 +84,7 @@ export const controlledCdrVersion = controlledCdrVersionTier.versions[0];
 
 export class CdrVersionsApiStub extends CdrVersionsApi {
   constructor() {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.